includes/discovery/sensors/ipmi.inc.php

```diff
@@ -1,47 +1,50 @@
 <?php
 
+use LibreNMS\Config;
+
 // IPMI - We can discover this on poll!
 if ($ipmi['host'] = get_dev_attrib($device, 'ipmi_hostname')) {
     echo 'IPMI : ';
 
     $ipmi['user']     = get_dev_attrib($device, 'ipmi_username');
     $ipmi['password'] = get_dev_attrib($device, 'ipmi_password');
 
-    if ($config['own_hostname'] != $device['hostname'] || $ipmi['host'] != 'localhost') {
+    if (Config::get('own_hostname') != $device['hostname'] || $ipmi['host'] != 'localhost') {
         $remote = " -H ".$ipmi['host']." -U '".$ipmi['user']."' -P '".$ipmi['password']."' -L USER";
     }
 
-    foreach ($config['ipmi']['type'] as $ipmi_type) {
-        $results = external_exec($config['ipmitool']." -I $ipmi_type".$remote.' sensor 2>/dev/null|sort');
+    foreach (Config::get('ipmi.type', array()) as $ipmi_type) {
+        $results = external_exec(Config::get('ipmitool')." -I $ipmi_type".$remote.' sensor 2>/dev/null|sort');
         if ($results != '') {
             set_dev_attrib($device, 'ipmi_type', $ipmi_type);
+            echo $ipmi_type;
```

Comment:
Maybe do `echo "$ipmi_type ";` so it will pop each one on a line for debug with a space otherwise it will be s single string of text.

Reply:
Oops, I must have moved this inside the loop for debug and never moved it back.  I'll move it back.